### ISSUE TITLE
`@enforce_keys` should not be treated as a regular module attribute

### ISSUE THREAD

**Issue by eksperimental** [rrrene/credo#1202]
This false positive happens with the following (simplified) config:

```ellixir
{Credo.Check.Readability.StrictModuleLayout,
 order: [
   :defstruct,
   :module_attribute,
 ]},
```

The Elixir code is:

```elixir
defmodule Foo do
  @enforce_keys [:bar]
  defstruct bar: nil
end
```

And the output of Credo is:

```
  Code Readability
┃
┃ [R] ↘ defstruct must appear before module attribute
```

So basically I will have to place `@enforce_keys` bellow `defstruct` which breaks the syntax.

Same care should be put with other special module attributes, if there is any other case.

**Comment by rrrene** [comment]
Thanks for reporting this 😀 This is, of course, a bug. 

It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by eksperimental** [comment]
Issue solved! Thank you for the extremely fast fix @rrrene
